lab-nassir/app/component/searchbar/searchbar.html

```diff
@@ -0,0 +1,4 @@
+<div class="searchbar">
+  <input type="text" ng-model="searchbarCtrl.searchTerm" placeholder="Search">
+  <div class="glyphicon glyphicon-remove-sign" ng-click="searchbarCtrl.clearSearch()"></div>
```

Comment:
Good job implementing a function to clear your search term! You could have also simply used an expression that set searchbarCtrl.searchTerm equal to null, but either way works.